packages/typespec-azure-resource-manager/test/resource.test.ts

```diff
@@ -883,3 +883,76 @@ it("recognizes resource with customResource identifier", async () => {
 `);
   expectDiagnosticEmpty(diagnostics);
 });
+
+describe("typespec-azure-resource-manager: identifiers decorator", () => {
```

Comment:
We should have another test here, where an array of models with a 'name' or 'id' property do not thrown any diagnostic, as 'name' and 'id' are automatically used if no x-ms-identifiers directive is present.

Reply:
On second thought, we have this test, so I think we are ok